armi/reactor/reactors.py

```diff
@@ -435,12 +435,26 @@ def removeAssembly(self, a1, discharge=True):
         else:
             self._removeListFromAuxiliaries(a1)
 
-    def removeAssembliesInRing(self, ringNum):
+    def removeAssembliesInRing(self, ringNum, forceHexRing=False):
         """
         Removes all of the assemblies in a given ring
+
+        Parameters
+        ----------
+        ringNum : int
+            The ring to remove
+
+        forceHexRing : bool, optional
+            If you want to remove a circular ring from a square or hex reactor.
```

Comment:
As a user, I find this sentence confusing. It sounds like based on this sentence that "True" would result in a circular ring being removed from a square or hex reactor which conflicts with the actual function and the variable name. Maybe use instead, "If you want to remove assemblies in square or hex ring from a squre or hex reactor. False removes assemblies in a circular ring"

Reply:
Fixed